app/services/mods_validator.rb

```diff
@@ -1,9 +1,11 @@
 # frozen_string_literal: true
 
 class ModsValidator
-  SCHEMA = 'app/helpers/xml/mods-3-6.xsd'
+  SCHEMA = 'mods-3-6.xsd'
```

Comment:
Please revert this and the choir. They are not necessary

Reply:
I think this required. Initially I didn't change any code in `mods_validator.rb` class but it failed both the build and when Andrew tested on stage.

Reply:
@jermnelson I think this is because we haven't specified the full paths to the schemas in the XML, so